docs/.vuepress/theme/Layout.vue

```diff
@@ -36,9 +36,17 @@
     },
     mounted () {
       window.addEventListener('scroll', this.onScroll)
-      if (localStorage) {
-        this.darkMode = localStorage.getItem('dark-mode') || false
+      if (localStorage.getItem('dark-mode') !== null) {
+        this.darkMode = localStorage.getItem('dark-mode') === "true" 
       }
+      else {
+        this.darkMode = matchMedia('(prefers-color-scheme: dark)').matches
+      }
+      window.matchMedia('(prefers-color-scheme: dark)').addListener(({ matches }) => {
```

Comment:
Is this necessary? What's the purpose?

Reply:
If you change the color scheme the website will dynamically change with it. It's not superfluous.